recipes/umi_tools/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
 test:
   commands:
     - umi_tools --help
+    - test -f $SP_DIR/umi_tools/_dedup_umi.c
```

Comment:
well the confusing thing is: it does work on my mac. But I have conda `conda 4.3.18`, maybe bioconda's version is older?